app/graphql/types/profile.rb

```diff
@@ -38,30 +38,32 @@ class Profile < Types::BaseObject
 
     field :compliant_host_count, Int, null: false
     def compliant_host_count
-      object.hosts.count { |host| object.compliant?(host) }
+      object.hosts.in_inventory.count { |host| object.compliant?(host) }
     end
 
     def total_host_count
-      object.hosts.count
+      object.hosts.in_inventory.count
     end
 
     def compliant(system_id:)
-      object.compliant?(Host.find(system_id))
+      object.compliant?(Host.in_inventory.find(system_id))
```

Comment:
Not a huge fan of `default_scope` for anything that's not sorting or similar (https://www.ombulabs.com/blog/ruby/rails/best-practices/why-using-default-scope-is-a-bad-idea.html) - I prefer to be explicit about this since it's key for our app